ui/component/publishForm/view.jsx

```diff
@@ -260,13 +261,13 @@ function PublishForm(props: Props) {
   React.useEffect(() => {
     if (activeChannelClaimStr) {
       const channelClaim = JSON.parse(activeChannelClaimStr);
-      const message = 'get-claim-id-replays';
```

Comment:
I think we want to show both legacy and new replays, so would this break legacy replays?
I'm just guessing here, so could be wrong.

Reply:
Legacy replays still work without signatures, so maybe a bad signature works too.